osgar/drivers/oak_camera.py

```diff
@@ -93,9 +94,12 @@ def __init__(self, config, bus):
             self.mono_resolution = g_resolution_dic[self.mono_resolution]
 
         color_resolution_value = config.get("color_resolution", "THE_1080_P")
-        assert color_resolution_value in["THE_1080_P", "THE_4_K", "THE_12_MP", "THE_13_MP"], color_resolution_value
         self.color_resolution = getattr(dai.ColorCameraProperties.SensorResolution, color_resolution_value)
 
+        color_orientation = config.get("color_orientation", "AUTO")
+        assert color_orientation in ["AUTO", "HORIZONTAL_MIRROR", "NORMAL", "ROTATE_180_DEG", "VERTICAL_FLIP"]
```

Comment:
here I would usually write
assert color_orientation in ["AUTO", "HORIZONTAL_MIRROR", "NORMAL", "ROTATE_180_DEG", "VERTICAL_FLIP"], color_orientation
(i.e. to see the used value in assert, but feel free to ignore it)

Reply:
ok, thanks